cclib/method/population.py

```diff
@@ -32,6 +34,13 @@ def __repr__(self):
         """Return a representation of the object."""
         return "Population"
 
+    def _check_required_attributes(self):
+        super(Population, self)._check_required_attributes()
+        
+        if not any(hasattr(self.data, a) for a in self.overlap_attributes):
+            raise MissingAttributeError(
+                    'Need overlap matrix (aooverlaps or fooverlaps attribute) for Population methods')

```

Comment:
The overlap matrix isn't required for CSPA.

Reply:
Indeed! Done in 83c0738ef18535f16e807a1f5235329e54cf6a3b.